src/BrightLocal/Api.php

```diff
@@ -134,4 +130,36 @@ public function delete($method, $params = array()) {
     public function get_last_http_code() {
         return $this->lastHttpCode;
     }
+
+    /**
+     * @param $httpMethod
+     * @param array $params
+     * @return array
+     */
+    private function get_options(string $httpMethod, array $params):array {
```

Comment:
Please don't use PHP 7 syntax in the wrapper. We should use PHP 5.6 compatible code.